strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/io/ArtifactOutputStreamTest.java

```diff
@@ -42,13 +42,22 @@
 {
 
     public static final String REPOSITORY_RELEASES = "aos-releases";
+    
+    private static boolean initialized;
```

Comment:
Please review https://youtrack.carlspring.org/issue/SB-757

`AccessDeniedException` problem was associated with indexer usage, and unfortunate double  `RepositoryIndexerFactory.createRepositoryIndexer(String, String, String, File, File)`   method invocation.

`initialized` was a solution for` java.nio.channels.OverlappingFileLockException` and the reason was 
> @PostConstruct is being called before every test method

The same trick was a solution for many other tests.